src/flow/index.js

```diff
@@ -19,6 +22,7 @@ const Flow = ({ data, config }) => {
 Flow.propTypes = {
   data: PropTypes.object,
   config: PropTypes.object,
+  dimension: PropTypes.object.isRequired,
```

Comment:
if this is intended to have `useWindowDimensions()` as default, shouldn't this be not required and get its default props `{ dimension: useWindowDimensions() }`?